backend/OpenAPI.yaml

```diff
@@ -198,7 +198,7 @@ paths:
             schema:
               $ref: "#/components/schemas/UserGetResponse"
       responses:
-        "201":
+        "200":
           description: User updated successfully
```

Comment:
The description also has to be changed. This endpoint shouldn't be performing any updates.